### PR TITLE
added about and conduct pages

### PR DESCRIPTION
**Pull request by tjnel**
I have created some place holder pages
- about
- Code of Conduct (using the django Code of conduct)

### CODE REVIEW

**Comment by tjnel** [comment]
This should address #23

**Comment by sgregg85** [comment]
The about page says that we meet at 9PM PST instead of EST, besides that, everything else looks good

**Comment by tjnel** [comment]
Fixed the time to 6PM/PST

File: content/pages/about.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Title: About
+Date: 2016-10-23
+
+Python Monthly is a online python community that meets every first Thursday of the month at 6PM/PST via the [Python Monthly YouTube Channel](https://www.youtube.com/channel/UC7ROIuBkGtLpCIMsgoIwv0Q). Each month we will offer presentations and lightning style talks for various python topics. We will also participate in community projects. Python monthly is a open community with members of all skill levels.
```
Comment (aculler) [inline content/pages/about.md:4]: Pretty sure Python is a proper noun and should be capitalized.

File: content/pages/codeofconduct.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+* **Be considerate**: Your work will be used by other people, and you in turn will depend on the work of others. Any decision you take will affect users and colleagues, and you should take those consequences into account when making decisions. Remember that we're a world-wide community, so you might not be communicating in someone else's primary language.
+* **Be respectful**:  Not all of us will agree all the time, but disagreement is no excuse for poor behavior and poor manners. We might all experience some frustration now and then, but we cannot allow that frustration to turn into a personal attack. It’s important to remember that a community where people feel uncomfortable or threatened is not a productive one.
+* **Be careful in the words that you choose**: we are a community of professionals, and we conduct ourselves professionally. Be kind to others. Do not insult or put down other participants. Harassment and other exclusionary behavior aren't acceptable. This includes, but is not limited to:
```
Comment (aculler) [inline content/pages/codeofconduct.md:10]: Capitalize the first 'W'?

**Review by aculler — CHANGES_REQUESTED** [review]
I didn't do an in-depth review of the HTML files since they're generated by Pelican. Just a few minor grammatical fixes and it'll be all set for merging.

**Pull request merged**